src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -739,6 +739,7 @@ static inline int MPIDI_NM_mpi_init_hook(int rank,
     /* ------------------------------------------------------------------------ */
     memset(&cntr_attr, 0, sizeof(cntr_attr));
     cntr_attr.events = FI_CNTR_EVENTS_COMP;
+    cntr_attr.wait_obj = FI_WAIT_UNSPEC;
```

Comment:
Should this also be added here?

https://github.com/pmodels/mpich/blob/115b7c7ab5e052eae25733445d7e92509e0e3fc3/src/mpid/ch4/netmod/ofi/ofi_win.h#L193

Reply:
Oh, Sure, this have to be updated. Per win synchronization has the same problem as ep-counter.
Thanks!